rotkehlchen/chain/ethereum/defi/zerionsdk.py

```diff
@@ -137,6 +138,16 @@
     'TimeWarp • Staking',
     'Gelato Network • Locked GEL',
 )
+WEIGHTED_BALANCE_NODES = (
```

Comment:
Rename this. We will forget about why these nodes are singled out. Specify somehow in the name that they can handle big gas limit calls.